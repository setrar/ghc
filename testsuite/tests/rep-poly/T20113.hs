{-# LANGUAGE PolyKinds, UnliftedNewtypes, NoFieldSelectors #-}
import GHC.Exts

newtype Y (a :: TYPE rep) = MkY { y_fld :: a }
