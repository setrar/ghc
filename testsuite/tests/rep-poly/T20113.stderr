
T20113.hs:4:35: error:
    The type of the first pattern in the equation for ‘$sel:y_fld:MkY’
    does not have a fixed runtime representation:
      Y a :: TYPE rep
